kong/cmd/utils/prefix_handler.lua

```diff
@@ -706,6 +706,19 @@ local function prepare_prefix(kong_config, nginx_custom_template_path, skip_writ
   end
   pl_file.write(kong_config.nginx_kong_stream_conf, nginx_kong_stream_conf)
 
+  local nginx_conf, err = compile_nginx_conf(kong_config, nginx_template)
```

Comment:
I guess this re-ordering is because we are using the emitted file from the previous step. We should document it with a line of comment here.

Reply:
that's right. Agreed.